### PR TITLE
Fixed responses and repetions

### PR DESCRIPTION
**Pull request by gtfr**
The commits contain several fixes:
- #92
- #89
- Another issue: repetitions of yearly events does not work

### CODE REVIEW

File: public/event.less [hunk 1]
```diff
@@ -140,7 +140,7 @@
     padding: 0;
   }
 
-  .plugin-calendar-event-editor-dates {
+  .plugin-calendar-event-editor-dates .plugin-calendar-event-editor-date {
```
Comment (pitaj) [inline public/event.less]: What does this change make it look like?

Comment (gtfr) [inline public/event.less]: This is my solution to #92. I am not 100% sure if this is the correct/best way to go, but it fixed it in my environment!

Comment (pitaj) [reply public/event.less]: I'm asking for a a screenshot

Comment (gtfr) [reply public/event.less]: This is how it looks before the change:
before

And this is after the change:
after

Comment (pitaj) [reply public/event.less]: That's what I was fearing. I want it to be side-by-side on desktop.

**Comment by pitaj** [comment]
I've implemented fixes myself, thanks for the help though.

**Pull request closed without merging**